packages/components/radio/stories/radio.stories.js

```diff
@@ -0,0 +1,17 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.radio';
+
+storiesOf('ts-radio', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const label1 = text('label1', 'Red pill');
+		const label2 = text('label2', 'Blue pill');
+		const disabled = boolean('Disabled', false);
+
+		return html`
+			<ts-radio ?disabled="${disabled}" label="${label1}"></ts-radio>
```

Comment:
please add all available properties and  make it more dynamic

Reply:
Radio group has properties, like "value" and "index" which meant to be read-only. Added 'title' and 'focused' properties. How can we make more dynamic? Ability to customize the number of radio inputs?